src/backend/distributed/deparser/citus_ruleutils.c

```diff
@@ -305,7 +306,8 @@ pg_get_sequencedef(Oid sequenceRelationId)
  */
```

Comment:
You can extend this function description to include what different values of `includeIdentityDefaults` are doing.

Reply:
Actually it would be good to combine this with the `includeSequenceDefaults` and explain in which cases we are using them as well: 
- in `alter_distributed_table path we use `NEXTVAL_SEQUENCE_DEFAULTS` and `INCLUDE_IDENTITY_DEFAULTS`,
- when creating the shell table in the workers, we use `WORKER_NEXTVAL_SEQUENCE_DEFAULTS` and `INCLUDE_IDENTITY_AS_SEQUENCE_DEFAULTS`,
- when we propagate to shards, we use `NO_SEQUENCE_DEFAULTS` and `NO_IDENTITY`.
- any other subcase we have?

A comment encapsulating the above would be really useful.